examples_tests/31.SkinningDataBenchmark/2.vert

```diff
@@ -0,0 +1,26 @@
+#version 460 core
+
+layout( push_constant, row_major ) uniform Block 
+{
+	mat4 vp;
+}pushConstants;
+
+layout(std430, set = 0, binding = 0, row_major) restrict readonly buffer BoneMatrices
+{
+    mat4x3 matrix[];
+};
+
+layout(location = 0) in vec3 pos;
+layout(location = 3) in vec3 normal;

```

Comment:
disable these inputs when benchmarking ( you can make the `pos = vec4(4,3,2,1)` a constant)